rustfmt.toml

```diff
@@ -0,0 +1,10 @@
+format_code_in_doc_comments = true
```

Comment:
I want to use rustfmt at factory setting always, so please don't.

Reply:
will remove